src/cfnlint/rules/parameters/AllowedValue.py

```diff
@@ -28,12 +28,11 @@ class AllowedValue(CloudFormationLintRule):
     source_url = 'https://github.com/awslabs/cfn-python-lint/blob/master/docs/cfn-resource-specification.md#allowedvalue'
     tags = ['resources', 'property', 'allowed value']
 
-    def __init__(self, ):
-        """Init """
+    def initialize(self, cfn):
         super(AllowedValue, self).__init__()
-        for resource_type_spec in RESOURCE_SPECS.get('us-east-1').get('ResourceTypes'):
+        for resource_type_spec in RESOURCE_SPECS.get('eu-west-1').get('ResourceTypes'):
```

Comment:
you mean `cfn.regions[0]`?

Reply:
Yup, I do;... Changes.

I will fix this in a more "structural" manner using helper functions